Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -479,6 +521,19 @@ namespace WhiteBox
         m_defaultModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "SketchMode");
         m_edgeRestoreModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "RestoreMode");
 
+        // temporary setting to disable this feature
+        if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get())
+        {
+            bool hasTransformMode = false;
+            if(!settingsRegistry->Get(hasTransformMode, WhiteBoxTransformFeature)) {
+                hasTransformMode = false;
+            }
```

Comment:
nit: Please always have braces start on a new line - I'd recommend using `clang-format` to do this, but I think White Box used a slightly different style (120 character limit instead of 140). So may run a formatting pass as a separate formatting only change to make things simpler in future.

Reply:
Need to remember to run clang format

Reply:
Just a quick FYI be careful with this as you may get more changes than you expect, so it might be worth doing this as a separate pass, just something to be aware of

Reply:
Ah nice! That's a great idea 🙂 Well remembered 👍